opbeans/src/main/java/co/elastic/apm/opbeans/model/ProductType.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * #%L
+ * Opbeans Java Demo Application
+ * %%
+ * Copyright (C) 2018 the original author or authors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package co.elastic.apm.opbeans.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@Entity(name="product_types")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class ProductType {
+	
+	@Id
+    @GeneratedValue
```

Comment:
nit: use tab

Reply:
ouch. I'm using STS and I didn't change the code style, will have a look.

Reply:
STS defaults to tabs 😞